modules/rosa-cluster-autoscaler-cli-describe.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-cluster-autoscaling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-cluster-autoscaler-cli-describe_{context}"]
+== View autoscaler configurations with the ROSA CLI
+
+You can view your cluster autoscaler configurations using the `rosa describe autoscaler` command.
+
+//ROSA HCP procedure
+ifdef::openshift-rosa-hcp[]
+.Procedure
+
+. To view cluster autoscaler configurations, run the following command:
```

Comment:
Both of the ROSA Classic and HCP procedures are single steps. Should these be unordered lists?

Reply:
Yes!